app/templates/meetings/edit/treatment.hbs

```diff
@@ -0,0 +1,35 @@
+<AuModal @modalOpen={{true}}
+         @size="fullscreen"
+         @closeModal={{this.closeModal}}
+         @modalTitle={{this.document.title}}
+         class="au-c-modal--editor"
+         as |Modal|
+>
+  <Modal.Body>
+    {{#if this.document}}
+      <BesluitDocumentContainer @document={{this.document}}
+                                @handleRdfaEditorInit={{this.handleRdfaEditorInit}}
+                                @editor={{this.editor}}
+                                @saving={{this.saveDocumentTask.isRunning}}
+      />
+    {{/if}}
+  </Modal.Body>
+  <Modal.Footer>
+    <AuButtonGroup>
+      <AuButton @loading={{this.saveDocumentTask.isRunning}}
+        {{on "click" (perform this.saveDocumentTask)}}>
+        {{t "behandelingVanAgendapunten.saveButton"}}
+      </AuButton>
+      <AuButton @loading={{this.saveDocumentTask.isRunning}} {{on "click" this.saveAndQuit}}>
+        {{t "behandelingVanAgendapunten.saveAndQuit"}}
+      </AuButton>
+    </AuButtonGroup>
+    <AuButtonGroup>
+       <AuButton @skin="secondary"
+                @loading={{this.saveDocumentTask.isRunning}}
+        {{on "click" this.cancel}}>
```

Comment:
Any specific reason why you split `cancel` and `closeModal` up into separate actions?

Reply:
I believe the technical term for it is "being an idiot"